RFCs/Template.md

```diff
@@ -0,0 +1,34 @@
+# RFC-#: Title
+
+**To be Reviewed By:** 
+
+**Authors:**
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+*What is this proposal solving? Why does the problem matter? How does the problem affect the user?*
+
+### Anti-Goals
+*What is outside the scope of what the proposal is trying to solve?*
+
+## Solution
+*Describe your solution and how it’s going to solve the problem. This is likely the largest section of your proposal and might even include some high-level diagrams if you are proposing code changes. While all important aspects need to be covered, also keep in mind that shorter documents are more likely to be read.*
+
+### User Impact
+*If you are proposing to add or modify public interfaces, those changes should be outlined here in detail.*
+
+### Backwards Compatibility and Upgrade Path
+*Will the regular rolling upgrade process work with these changes?*
+
+*How do the proposed changes impact backwards-compatibility? Are message or file formats changing?*
+
+*Is there a need for a deprecation process to provide an upgrade path to users who will need to adjust their applications?*
+
+## FAQ
+*Answers to questions you’ve commonly been asked after requesting comments for this proposal.*
```

Comment:
Should be ok to include FAQs that came up while writing the proposal too.

Reply:
Yes. updated text to: `Answers to common questions that may arise and those that you’ve commonly been asked after requesting comments for this proposal.`